modules/persistent-storage-csi-gcp-filestore-nfs-export-options.adoc

```diff
@@ -0,0 +1,49 @@
+// Module included in the following assemblies:
+//
+// * storage/container_storage_csi-google_cloud_file.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="persistent-storage-csi-gcp-filestore-nfs-export-options_{context}"]
+= NFS export options
+
+Network File System (NFS) sets file share permissions when creating a storage class. Using NFS export options propagate these options to the server for the Filestore instance.
+
+.Prerequisites
+* Access to the cluster as a user with the cluster-admin role.
+
+* The {gcp-short} Filestore CSI Driver Operator and {gcp-short} Filestore CSI driver installed.
+
+.Procedure
+
+. Create a storage class using a file similar to the following sample YAML file:
++
+[NOTE]
+====
+For more information about creating a storage class, see Section _Creating a storage class for GCP Filestore Operator_.
+====
++
+.Example storage class YAML file with NFS export options
+[source,yaml]
+----
+kind: StorageClass
+apiVersion: storage.k8s.io/v1
+metadata:
+ name: SC-name
+provisioner: filestore.csi.storage.gke.io
+parameters:
+ connect-mode: DIRECT_PEERING
+ network: project-network
+ nfs-export-options-on-create: '[ <1>
+   {
+     "accessMode": "READ_WRITE", <2>
+     "squashMode": "NO_ROOT_SQUASH", <3>
+     "ipRanges": [ <4>
+       "10.0.0.0/16"
+     ]
+   }]'
+allowVolumeExpansion: true
+----
+<1> *NFS export options parameter*: `nfs-export-options-on-create` must start with `[` and end with `}]`, otherwise YAML cannot parse it. Also, parameters must be in ​​quotations marks, otherwise a parsing error occurs during persistent volume claim (PVC) creation.
```

Comment:
`otherwise YAML cannot parse it` -> this is parsed as JSON, not YAML: https://github.com/kubernetes-sigs/gcp-filestore-csi-driver/pull/737/files#diff-824bc5c10ceb462bc5f3e0fa91ebc1930d717bd471bb9403937e902a6b4550d3R1028

`must start with `[` and end with `}]`` -> this is misleading, we can say the parameters have to be in `{` `}` to create a dictionary/map which holds the parameters as key value pairs and is enclosed with single `[` `]` (to form a list), but it should be apparent from the example and we can mention it's JSON format.

Reply:
I'm inclined to remove this. It is clear from the example, and I feel like we'd be over explaining to go into the rest.